id: PYSEC-2020-10
package:
  name: ansible
  ecosystem: PyPI
summary: Vulnerability in ansible
details: A flaw was found in Ansible Engine when the module package or service is
  used and the parameter 'use' is not specified. If a previous task is executed with
  a malicious user, the module sent can be selected by the attacker using the ansible
  facts file. All versions in 2.7.x, 2.8.x and 2.9.x branches are believed to be vulnerable.
affects:
  ranges:
  - type: ECOSYSTEM
    fixed: 2.7.17
  - type: ECOSYSTEM
    introduced: 2.8.0
    fixed: 2.8.9
  - type: ECOSYSTEM
    introduced: 2.9.0
    fixed: 2.9.6
references:
- type: WEB
  url: https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1738
- type: WEB
  url: https://github.com/ansible/ansible/issues/67796
- type: WEB
  url: https://security.gentoo.org/glsa/202006-11
aliases:
- CVE-2020-1738
modified: "2020-06-13T04:15:00Z"
published: "2020-03-16T16:15:00Z"
